webclient/dokument/models.py

```diff
@@ -519,6 +524,35 @@ def generate_redis_snapshot(self):
             data = RedisConnector.prepare_model_for_redis(table)
             return self.redis_snapshot_id, data
 
+    def get_komponenty(self, arch_z_status=None):
```

Comment:
Asi bych to prejmenoval na get komponenty for status alebo except status

Reply:
Ten parametr `arch_z_status` je nepovinný, tak nevím, jestli by to spíše nebylo matoucí.